src/diffusers/models/attention.py

```diff
@@ -506,7 +513,9 @@ def forward(
             norm_hidden_states = norm_hidden_states * (1 + scale_mlp[:, None]) + shift_mlp[:, None]
 
         if self.norm_type == "ada_norm_single":
-            norm_hidden_states = self.norm2(hidden_states)
+            norm_func = self.norm2 if self.norm2 is not None else self.norm3
+            # norm_hidden_states = self.norm2(hidden_states)
```

Comment:
Can be removed.

Reply:
Hmm a little weird. `norm1`, `norm2`, and `norm3` all feel very sequential in nature to me. So, I don't understand how `norm3` could be present without the presence of `norm2`. It's a subtle point but it feels very unnatural to me.

Reply:
In the Latte temporal Transformer block, only one multi-head attention exists. Therefore, We have `norm1`, `attn1`. But we don't have `norm2`, `attn2` (`attn2` is usually used cross attention between text and image). `norm3` is used for `MLP`.